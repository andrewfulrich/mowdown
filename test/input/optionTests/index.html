<html>
  <script src='/jsScript.js'></script>
  <script src='/jsScript2.js'></script>
  <link rel="stylesheet" type="text/css" href="/cssScript.css">
  <link rel="stylesheet" type="text/css" href="/cssScript2.css">
</html>