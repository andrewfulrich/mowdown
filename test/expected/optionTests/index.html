<html><head><script src="https://cdn.jsdelivr.net/npm/vue"></script><script src="/bundle-index.js"></script><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"><link rel="stylesheet" href="/index.css"></head><body></body></html>